Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -76,6 +89,44 @@ private Task AddProjectsToSolution()
                 new[] { projectInfo, projectTestInfo });
         }
 
+        private Task AddInternalNugetServerToNugetConfig()
+        {
+            var nugetConfigFileName = "NuGet.config";
```

Comment:
make this a constant

Reply:
Addressed in: [Make variable const](https://github.com/messerli-informatik-ag/meta-generator/pull/22/commits/8c5bdf4b730639832151dbbe7f46567e33bc5d4a)